README.md

```diff
@@ -1,107 +1,154 @@
-# Liquity System Summary
+# Liquity: Decentralized Borrowing Protocol
 
 ![Tests](https://github.com/liquity/dev/workflows/CI/badge.svg) [![Frontend status](https://img.shields.io/uptimerobot/status/m784948796-056b56fd51c67d682c11bb24?label=Testnet&logo=nginx&logoColor=white)](https://devui.liquity.org) ![uptime](https://img.shields.io/uptimerobot/ratio/7/m784948796-056b56fd51c67d682c11bb24) [![Discord](https://img.shields.io/discord/700620821198143498?label=join%20chat&logo=discord&logoColor=white)](https://discord.gg/2up5U32) [![Docker Pulls](https://img.shields.io/docker/pulls/liquity/dev-frontend?label=dev-frontend%20pulls&logo=docker&logoColor=white)](https://hub.docker.com/r/liquity/dev-frontend)
 
+
+Liquity is a decentralized protocol that allows Ether holders to obtain maximum liquidity against
+their collateral without paying interest. After locking up ETH as collateral in a smart contract and
+creating an individual position called a "trove", the user can get instant liquidity by minting LUSD,
+a USD-pegged stablecoin. Each trove is required to be collateralized at a minimum of 110%. Any
+owner of LUSD can redeem their stablecoins for the underlying collateral at any time. The redemption
+mechanism along with algorithmically adjusted fees guarantee a minimum stablecoin value of USD 1.
+
+An unprecedented liquidation mechanism based on incentivized stability deposits and a redistribution
+cycle from riskier to safer troves provides stability at a much lower collateral ratio than current
+systems. Stability is maintained via economically-driven user interactions and arbitrage, rather
+than by active governance or monetary interventions.
+
+The protocol has built-in incentives that encourage both early adoption and the operation of
+multiple front ends, enhancing decentralization.
+
+## More information
+
+Visit [liquity.org](https://www.liquity.org) to find out more and join the discussion.
+
+### Packages
```

Comment:
This section is a bit of a duplicate of https://github.com/liquity/dev#project-structure

Reply:
Good spot - fixed: https://github.com/liquity/dev/pull/529/commits/40f7f5f64fbda7eaa2b2141e4902358869ecb194